fireAlert.cpp

```diff
@@ -0,0 +1,18 @@
+#include "fireAlert.h"
+#include<string.h>
+
+using namespace std;
+
+void fireAlert(AllMessages *messages, Alerter *alerters)
+{
+    for (int i = 0; i < numberOfInputVitals; i++)
+    {   
+        if (strcmp(messages[i].message, "is ok"))
+        {
+            for (int j = 0; j < numberOfInputAlerts; j++)
```

Comment:
The inner for loop over here can be in another function called something like 'triggerAllAlerters'.
This function can be used anywhere you need to trigger all of them.